app/controllers/contacts_controller.rb

```diff
@@ -0,0 +1,58 @@
+class ContactsController < ApplicationController
+  before_action :set_contact, only: [:show, :edit, :update, :destroy]
+
+  def index
+    @contacts = Contact.all
+  end
+
+  def show; end
+
+  def new
+    @contact = Contact.new
+  end
+
+  def edit; end
+
+  def create
+    @contact = Contact.new(contact_params)
+
+    respond_to do |format|
```

Comment:
Simplify to
```ruby
if @contact.save
  redirect_to @contact, notice: t('contact_created')
else
  render :new
end
```
The whole `respond_to` is not needed, when we only respond to html.